lib/App.js

```diff
@@ -1676,12 +1701,30 @@ $ sudo systemctl restart docker
       await writeFileAsync(path.join(driverPath, 'driver.settings.compose.json'), JSON.stringify(driverWireless.settings, false, 2));
     }
 
+    const templatePath = path.join(__dirname, '..', 'assets', 'templates', 'app', 'drivers');
+
     // Matter devices do not need a device/driver file. As apps cannot add functionality to Matter devices.
     if (!driverWireless.matter) {
-      const fileExtension = App.usesTypeScript({ appPath: this.path }) ? '.ts' : '.js';
-      const templatePath = path.join(__dirname, '..', 'assets', 'templates', 'app', 'drivers');
-      await copyFileAsync(path.join(templatePath, `driver${fileExtension}`), path.join(driverPath, `driver${fileExtension}`));
-      await copyFileAsync(path.join(templatePath, `device${fileExtension}`), path.join(driverPath, `device${fileExtension}`));
+      if (App.usesModules({ appPath: this.path })) {
```

Comment:
Sluiten `type: module` en `typescript` elkaar nu uit?

Reply:
Of dwingen we af dat typescript altijd `.ts` moet zijn en nooit `.mts` mag zijn?

Reply:
We hebben sowieso nog geen support voor .mts